packages/ra-auth-auth0/package.json

```diff
@@ -17,8 +17,8 @@
     "peerDependencies": {
         "@auth0/auth0-spa-js": "^2.0.0",
         "ra-core": "^4.10.0",
```

Comment:
Can you please replaces all direct dependencies to `ra-core` by dependencies to `react-admin`?

Also, you need to use v5.

Reply:
I'm not sur we should do it. @fzaninotto change it in v1.1.0
cf https://github.com/marmelab/ra-auth-auth0/blob/main/CHANGELOG.md#110

Reply:
@fzaninotto only rewrote the changelog, but the actual change can be tracked back to a commit by @djhi : https://github.com/marmelab/ra-auth-auth0/commit/1595814616e96dc042f72af5c3e71d7023d92a8f#diff-34c62cae6b5d6d2d2e7a38bbb31cbd58180975ed57c7a158be7d8d3192e0ffa7

IMO we should still do it. I see no good reason to have `ra-core` and not `react-admin`, added to the fact that, [at the beginning](https://github.com/marmelab/ra-auth-auth0/blob/307909c59079a8400bc0107481907d510c4b9d8c/packages/ra-auth-auth0/package.json), we had both.